addons/portal/wizard/portal_wizard.py

```diff
@@ -164,6 +164,7 @@ def _create_user(self):
             'email': extract_email(self.email),
             'login': extract_email(self.email),
             'partner_id': self.partner_id.id,
+            'name': self.partner_id.name,
```

Comment:
This is not needed anymore as it was fixed with https://github.com/odoo/odoo/commit/50b893474537ceffa1ff83465c08e0ec9bbc99fc

Reply:
reverted